src/helpers/Environment.ts

```diff
@@ -99,13 +121,16 @@ export class EnvHelper {
    * @returns array of API urls
    */
   static getAPIUris() {
-    // Debug log
-    // console.log("uris", EnvHelper.getAlchemyAPIKeyList(), EnvHelper.getSelfHostedSockets());
     const ALL_URIs = [...EnvHelper.getAlchemyAPIKeyList(), ...EnvHelper.getSelfHostedNode()];
```

Comment:
since we want to live on the node as the only primary, REACT_APP_ALCHEMY_IDS should be blank or not present in the ENV